openspec/specs/cli-init/spec.md

```diff
@@ -86,12 +86,36 @@ This project uses OpenSpec to manage AI assistant workflows.
 The command SHALL provide an interactive menu for AI tool selection with clear navigation instructions.
 
 #### Scenario: Displaying interactive menu
-- **WHEN** run in fresh or extend mode
+- **WHEN** run in fresh or extend mode without non-interactive options
```

Comment:
If you have the spec deltas file above, you don't need this file!

When we archive the above change delta file it'll automatically update this file with the new requirements :)
You can remove the changes here!